### ISSUE TITLE
release 22.2.0: problem with checksum files

### ISSUE THREAD

**Issue by sebalis** [dbeaver/dbeaver#17682]
### Description

Hi, the content of https://dbeaver.io/files/22.2.0/checksum/dbeaver-ce_22.2.0_amd64.deb.sha256 does not match my locally calculated checksum for https://github.com/dbeaver/dbeaver/releases/download/22.2.0/dbeaver-ce_22.2.0_amd64.deb – the sha256 file contains 01deb3834681699775e28ec722ec509a1e275a69e62624c9612f47553f370c5f  and I get 00f5f29e380dc0898fda8d156f65b7bdb18596940e5fc2bb388d208cd6fed29a – and this used to match in many previous versions. Has something been changed in the way these files are distributed?

### DBeaver Version

22.2.0

### Operating System

Debian Linux

**Comment by jkjunior** [comment]
Same issue.

```
File has unexpected size (111471768 != 111472372). Mirror sync in progress? [IP: 104.131.190.124 443]
  Hashes of expected file:
   - SHA512:3fda9c99ccef233bb8097f13ea2938bf3dcdc69cd9d870622ac1d99498e03aaf015966ae943f0e0aaf0a0f74004e595a8996fd52f6ed3f90dfb2e4b9ba5fd7b0
   - SHA256:00f5f29e380dc0898fda8d156f65b7bdb18596940e5fc2bb388d208cd6fed29a
   - SHA1:2dc90741cadc3922e6f1756cf026b30bff265cbd [weak]
   - MD5Sum:a42c218879fb007796aab5f1c8053f47 [weak]
   - Filesize:111472372 [weak]
E: Failed to fetch https://dbeaver.io/debs/dbeaver-ce/./dbeaver-ce_22.2.0_amd64.deb  File has unexpected size (111471768 != 111472372). Mirror sync in progress? [IP: 104.131.190.124 443]
   Hashes of expected file:
    - SHA512:3fda9c99ccef233bb8097f13ea2938bf3dcdc69cd9d870622ac1d99498e03aaf015966ae943f0e0aaf0a0f74004e595a8996fd52f6ed3f90dfb2e4b9ba5fd7b0
    - SHA256:00f5f29e380dc0898fda8d156f65b7bdb18596940e5fc2bb388d208cd6fed29a
    - SHA1:2dc90741cadc3922e6f1756cf026b30bff265cbd [weak]
    - MD5Sum:a42c218879fb007796aab5f1c8053f47 [weak]
    - Filesize:111472372 [weak]
```

**Comment by sebalis** [comment]
A new version has been released (22.2.1) and this time the sha256 file at dbeaver.io matches the software file at Github. So the issue itself no longer exists. But I worry about the fact that this issue does not seem to have been noticed by anyone, judging by the lack of responses.

**Comment by sebalis** [comment]
I should rephrase that: The issue does still exist for version 22.2.0.

**Comment by serge-rider** [comment]
It was a result of emergency hotfix version release.
We didn't re-uploaded github artifacts at that time.
Files on dbeaver.io are ok.

I don';t think that we should patch the past for now..

**Comment by sebalis** [comment]
Yes, I can live with “the past remaining unpatched”, But this comment still helped me. I have an update script for dbeaver that downloads the .deb package and the checksum. I bleieve that when I wrote that script, the only place to download the .deb file was from Github (it’s possible that I overlooked something.) Now when I download this file from the website the download comes from dbeaver.com. I have now adapted my script to use ```https://download.dbeaver.com/community/${version}/dbeaver-ce_${version}_amd64.deb``` and I’m optmistic that in the future I am less likely to run into a discrepancy between that file and ```https://dbeaver.io/files/${version}/checksum/dbeaver-ce_${version}_amd64.deb.sha256```. So I’m happy that I didn’t close the issue myself and grateful for the response. :-)
